packages/dds/tree/src/feature-libraries/modular-schema/modularChangeFamily.ts

```diff
@@ -127,14 +130,15 @@ export class ModularChangeFamily
 	 * The returned `FieldChangeset`s may be a shallow copy of the input `FieldChange`s.
 	 */
 	private normalizeFieldChanges(
-		change1: FieldChange | undefined,
-		change2: FieldChange | undefined,
+		change1: FieldChange,
+		change2: FieldChange,
```

Comment:
Usages of `change1` and `change2` in this function can be updated to assume they are not undefined.
It's sad the compiler/linter doesn't catch that.